content/docs/08-user-management/04-saml-sso/05-palette-sso-with-onelogin.md

```diff
@@ -0,0 +1,219 @@
+
+<br />
+
+## Enable OIDC in Palette 
+17. You're done configuring OneLogin, now you can navigate back to Palette and fill in the **Client ID**, **Client Secret** and **Issuer URL** values. Also add the **groups** scope to the **Scopes** field. Without this Palette won't receive the group name. With everything filled in, you can now click **Enable**. 
+
+<br />
+
+  ![Full OIDC Config](/oidc-onelogin-images/oidc-full-palette.png)
+
+<br />
+
+## Creating Teams in Palette
+
+18. In order for Palette to map the groups configured in OneLogin, you need to create a Team in Palette. This Team name needs to match with the value that is returned for the groups configured in OneLogin. In this example we'll be creating an **Administrators** team. This also matches with the Administrators value that is assigned to each member of the Administrators group. In Palette, go to **Tenant Settings** --> **Users & Teams** --> **Teams** and select **Create team**. You only need to fill in the Team name and don't have to manually add members. This will happen automatically, because the users will belong to a certain group and that group is mapped to a team with certain permissions in Palette. After filling in the Team name, you can click *confirm*. 
+
+<br />
+
+  ![Create New Team](/oidc-onelogin-images/new-team.png)
+
+<br />
+
+19. Next you need to give the members of this team a set of permissions. In this example we will give all members *Tenant Admin* permissions, since this is the Administrators team. Of course, you can define these permissions fully to your liking. In order to set the correct permissions, you can select the team you just created (**Tenant Settings** --> **Users & Teams** --> **Teams** --> **Your team**), then select **Tenant Roles** --> **Add Tenant Role** and select the **Tenant Admin** box. This will auto-select all other boxes. 
+
+<br />
+
+  ![Add Tenant Role](/oidc-onelogin-images/tenant-admin-role.png)
+
+<br />
+
+Click **Confirm** and you should have a configuration similar to this. 
+
+<br />
+
+  ![Permissions](/oidc-onelogin-images/roles-full.png)
+
+<br />
+
+# Validate
+
+1. The SSO configuration is now complete and it's time to test all of our efforts. You can logout out of Palette and you'll be redirected to a new login screen. 
+
+<br />
+
+  ![Logout](/oidc-onelogin-images/logout.png)
+
+<br />
+
+2. The login screen now defaults to a **Sign In** button. If you are already signed in into OneLogin, once you click on **Sign In** you will be automatically signed in and should have the correct permissions set. If you are unable to sign in, you can always use a username and password. 
+
+<br />
+
+  ![SSO Login Screen](/oidc-onelogin-images/login-screen.png)
+
+<br />
+
+
+3. You can check which teams you've been mapped to, by going to **Tenant Settings** --> **Users & Teams** --> **Users** and you'll directly see which teams you belong to.  
+
+<br />
+
+  ![Check Team Member](/oidc-onelogin-images/team-member.png)
```

Comment:
This screenshot is easy to miss. Let's include the entire screen but blur out other names.  That way, the reader can see where in the Palette UI to see this.